### PR TITLE
refactor(odyssey-react): tokens for Table theme

### PR DESCRIPTION
**Pull request by andrewberg-okta**

### CODE REVIEW

File: packages/odyssey-react/src/components/Table/TableCell.module.scss [hunk 1]
```diff
@@ -12,32 +12,32 @@
...
-  color: var(--ColorTextHeader);
-  font-weight: var(--FontWeightHeader);
+  color: var(--HeaderTextColor);
+  font-weight: var(--HeaderFontWeight);
 }
 
 .headerCellCol {
-  border-block-start: var(--BorderWidthHeaderCol) var(--BorderStyleHeaderCol)
-    var(--BorderColorHeaderCol);
+  border-block-start: var(--HeaderCellColBorderWidth)
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Table/TableCell.module.scss:29]:
```suggestion
  border-block-start-width: var(--HeaderCellColumnBorderWidth);
  border-block-start-style: var(--HeaderCellColumnBorderStyle);
  border-block-start-color: var(--HeaderCellColumnBorderColor);
```

also will abbreviations like `Col` be problematic here? I think a long form name may be easier to reason about?

File: packages/odyssey-react/src/components/Table/TableSortButton.module.scss [hunk 2]
```diff
@@ -31,21 +31,21 @@
...
 .unsortedDirection {
   .sortIndicator {
-    color: var(--ColorIndicatorUnsorted);
+    color: var(--UnsortedSortIndicatorTextColor);
   }
 
   &:hover {
     .sortIndicator {
-      color: var(--ColorIndicatorUnsortedHover);
+      color: var(--UnsortedHoverSortIndicatorTextColor);
```
Comment (jeffbelser-okta) [inline packages/odyssey-react/src/components/Table/TableSortButton.module.scss:48]: oh boy, this one is fun.

To me, SortIndicator should come before Hover as it's a child part and Hover is the state?

**Review by jeffbelser-okta — COMMENTED** [review]
strong work 💪

**Review by jeffbelser-okta — APPROVED** [review]
generally looks good - would be nice to talk through names and have more granular border styles to match variables rather than use shorthand

File: packages/odyssey-react/src/components/Table/TableCell.module.scss (hunk 1 above)
Comment (andrewberg-okta) [reply packages/odyssey-react/src/components/Table/TableCell.module.scss]: addressed in edef7d7

File: packages/odyssey-react/src/components/Table/TableSortButton.module.scss (hunk 2 above)
Comment (andrewberg-okta) [reply packages/odyssey-react/src/components/Table/TableSortButton.module.scss]: 👍 addressed in edef7d7

**Pull request merged**